doc/sphinx/Doxyfile.in

```diff
@@ -68,7 +68,8 @@ PROJECT_LOGO           =
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = "_build"
+OUTPUT_DIRECTORY       = _build
+# OUTPUT_DIRECTORY       = @CMAKE_CURRENT_BINARY_DIR@/sphinx/
```

Comment:
Do we need to keep this comment ?

Reply:
And add todo target in Doxyfile.in to fix this.

Reply:
You can remove the comment. Will fix later. I will do some issue.

Reply:
Comment removed.